packages/components/src/EmptyState/EmptyState.tsx

```diff
@@ -66,6 +67,7 @@ export const EmptyState = ({
   id,
   illustrationType,
   variant = "informative",
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   layoutContext = "sidebarAndContent",
```

Comment:
Understand why we're pulling in out of the spread, but do we need a default if we're not going to use it?

Reply:
Nope we do not! I'll remove it :)